templates/worker/00-worker/baremetal/files/dhcp-dhclient-conf.yaml

```diff
@@ -0,0 +1,7 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/dhcp/dhclient.conf.template"
+contents:
+  inline: |
+    supersede domain-search "{{ .EtcdDiscoveryDomain }}";
+    prepend domain-name-servers {{`{{ .NonVirtualIP }}`}};
```

Comment:
also, NonVirtualIP is never defined it seems, and it looks like we're not making use of the `/usr/local/bin/non_virtual_ip` script above. Are we missing a patch?